sub_peers.js

```diff
@@ -129,7 +129,7 @@ TChannelSubPeers.prototype.chooseLinearPeer = function chooseLinearPeer(req) {
         var hostPort = hosts[i];
         var peer = self._map[hostPort];
         if (!req || !req.triedRemoteAddrs || !req.triedRemoteAddrs[hostPort]) {
-            var score = peer.scoreStrategy.getScore(req);
+            var score = peer.getScore(req);
```

Comment:
hmm, it looks to me like we could separate out the change to the relation ship of peer and strategy from the switch to range api... thouhts?